rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -100,8 +101,8 @@ def persist(self, model_dir):
         # type: (Text) -> Dict[Text, Any]
         file_name = self.name+".json"
         full_name = os.path.join(model_dir, file_name)
-        with io.open(full_name, 'w') as f:
-            f.write(json.dumps({"dimensions": self.dimensions}))
+        with io.open(full_name, 'w', encoding="utf-8") as f:
```

Comment:
No sorry, I think I was wrong.

the wrapping should happen around the `json.dumps` as `f.write(str(json.dumps(...)))`. (keeping `with io.open(full_name, 'w') as f:` instead of the version with unicode).